LICENSE.md

```diff
@@ -0,0 +1,2 @@
+LCD game JavaScript library
+Copyright Bas de Reuver (c) 2018
```

Comment:
This way you don't have to include the copyright notice with every file. I know it's standard in some companies, but this way is legally the same, AFAIK.